guides/common/modules/proc_retrieving-archived-metrics.adoc

```diff
@@ -3,9 +3,10 @@
 [id="retrieving-archived-metrics_{context}"]
 = Retrieving archived metrics
 
+[role="abstract']
 You can use the PCP CLI tools to retrieve metrics from an archive file.
 
-.Examples
+Here are some examples:
```

Comment:
This is currently a procedure module so Vale will report that it's missing a `.Procedure` heading.

However, looking at the contents, it's actually more like reference. So if you rename the file to `ref_...adoc`, change `PROCEDURE` to `REFERENCE` on line 1, you'll avoid that problem. And then you'll be able to drop the `Here are some examples:` line.